src/Filter/AlwaysOpenAutocomplete.types.tsx

```diff
@@ -0,0 +1,8 @@
+import { AutocompleteProps } from "@mui/material";
+
+export type AlwaysOpenAutocompleteProps = AutocompleteProps<
+  string,
+  boolean,
+  boolean,
+  boolean
+>;
```

Comment:
Would it be possible to type this better, so we know what props are `string` and which are `boolean`?

Reply:
The typing is what it is here from the MUI component. We're just setting types for the four generics. We can add a comment against each to explain what it relates to. You can also just hover over the MUI type and it will explain what each arg is.

<img width="1762" alt="image" src="https://github.com/user-attachments/assets/55f4541e-9d67-4294-8cf0-9300eff150f2">